Magento_Reward/web/template/cart/reward.html

```diff
@@ -7,28 +7,27 @@
         "
     >
         <span class="order-summary__label">
-            <a
-                data-bind="
-                    attr: {
-                        href: rewardPointsRemoveUrl
-                    }
-                "
+            <button
+                data-role="opc-remove-reward"
+                data-bind="click: removeRewardPoints"
                 class="
                     button
                     button--link
                     padding-left-0
                     padding-right-s
```

Comment:
form what I've checked, the original template has `action action-remove` classes. it's safer to keep the original classes in case they're used / will be used in the future for js selectors. if these classes don't cause any styling issues then I think it would be good to add them